src/pages/enterprise/statusbar.md

```diff
@@ -1,13 +1,33 @@
 ---
-title: Status Bar
+title: Statusbar
 template: enterprise-plugin
-version: 2.4.2
-minor: 2.4.X
+version: 2.5.0
+minor: 2.5.X
 ---
 
 The Status Bar plugin provides the ability to customize the appearance of the native mobile status bar, including setting visibility, background color, and more.
 
-<native-ent-install plugin-id="statusbar" variables=""></native-ent-install>
+<native-ent-install plugin-id="statusbar" variables="" capacitor="false"></native-ent-install>
```

Comment:
remove the capacitor attribute.  Decided to use `capacitor-slug="filesystem"` if incompatible. If it is, simply don't apply the attribute

Reply:
I know, this is previous to your changes

Reply:
BTW, I've updated it